level2/coco3/modules/boot_rom.asm

```diff
@@ -80,13 +89,17 @@ ExitOK              andcc     #^Carry             clear carry
 Uhoh                puls      u,y,x,a,b,cc,pc
 
 
-* this code executes at $4800
+* this code executes at RelAddy
 RelCode             equ       *
                     lda       #$4E                CC3 mode, MMU, 32K ROM
                     sta       $FF90
                     sta       $FFDE               ROM/RAM mode
 
 * Map ROM Blocks in
+                    ldd       $FFA6
+                    pshs      d
+                    ldd       $FFA4
+                    pshs      d
                     ldd       #$3C3D
```

Comment:
I see what you're doing here, but I suspect that many readers will not.

And since this is the kind of code that other 6x09 SBC builders
will need to muck with, some comments would help.

Reply:
Fair point. I'll add some comments.